<?xml version="1.0"?>
<launch>
<group ns="mcr_perception">
    <node pkg="mcr_perception_selectors" type="cavity_pose_publisher_node" name="cavity_pose_publisher" output="screen">
    <param name="loop_rate" type="double" value="10"/>
     </node>
     </group>
</launch>